src/autogluon/bench/eval/evaluation/benchmark_evaluator.py

```diff
@@ -157,8 +157,15 @@ def _load_results(
         return results_raw
 
     def load_results_raw(self, paths: list) -> pd.DataFrame:
-        paths = [path if is_s3_url(path) else self.results_dir_input + path for path in paths]
-        return pd.concat([pd.read_csv(path) for path in paths], ignore_index=True, sort=True)
+        dataframes = []
+        for path in paths:
+            path = path if is_s3_url(path) else os.path.join(self.results_dir_input, path)
+            dataframe = pd.read_csv(path)
+            dataframes.append(dataframe)
+        # Discarding extra folds
+        min_num_rows = min(len(df) for df in dataframes)
```

Comment:
What if there are multiple datasets in results file? `min()` will not do what it's intended right?